src/components/@shared/FormInput/TagsAutoComplete.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import CreatableSelect from 'react-select/creatable'
+import { OnChangeValue } from 'react-select'
+import { useField } from 'formik'
+import { InputProps } from '.'
+import { getTagsList } from '@utils/aquarius'
+import { chainIds } from 'app.config'
```

Comment:
is this intended to get the tags of all initially defined `chainIds`? Cause usually we base everything on the `chainIds` the user has selected, which this here would not get

Reply:
Yes, it is intended behaviour since the request was to generate the list of existing tags coming from mainnets. Using the user selection would require an additional filter to exclude the testnet chainIds.

Reply:
yep, as intended. No need to fetch for each chain, and no need to fetch tags from test chains.